### ISSUE TITLE
Scheduler constantly cycling days on "next day" hour

### ISSUE THREAD

**Issue by ADFairbanks** [mnemosyne-proj/mnemosyne#98]
I opened up the program to study and it kept reloading every single second and thus changing the card. It fixes when I change the "day_starts_at" to another number in "config.py". My guess is that one part of the scheduler is checking the hour with rounding and another part does it with truncating resulting in this strange behavior on a specific time of day. This happens even with starting on a blank database and clean installation.

This log snippet might make things a little clearer:

2018-11-04 02:37:15 : Program started : Mnemosyne 2.6.1 nt win32 TZ -7
2018-11-04 02:37:15 : Scheduler : SM2 Mnemosyne
2018-11-04 02:37:15 : Loaded database 0 0 0
2018-11-05 02:37:15 : Loaded database 0 -666 -666
2018-11-06 02:37:15 : Loaded database 0 -666 -666
2018-11-07 02:37:15 : Loaded database 0 -666 -666
2018-11-08 02:37:15 : Loaded database 0 -666 -666
2018-11-09 02:37:15 : Loaded database 0 -666 -666
2018-11-10 02:37:15 : Loaded database 0 -666 -666
2018-11-11 02:37:15 : Loaded database 0 -666 -666
2018-11-04 02:37:16 : Saved database 0 0 0
2018-11-04 02:37:16 : Loaded database 0 0 0
2018-11-05 02:37:16 : Loaded database 0 -666 -666
2018-11-06 02:37:16 : Loaded database 0 -666 -666
2018-11-07 02:37:16 : Loaded database 0 -666 -666
2018-11-08 02:37:16 : Loaded database 0 -666 -666
2018-11-09 02:37:16 : Loaded database 0 -666 -666
2018-11-10 02:37:16 : Loaded database 0 -666 -666
2018-11-11 02:37:16 : Loaded database 0 -666 -666

**Comment by pbienst** [comment]
I cannot reproduce this. With the default settings of day_starts_at, does this behaviour occur at all possible hours in the day?

The multiple 'loaded database' statements in the logs with dates in the past are harmless and unrelated, it's just a hack to be able to recover extra information about the historic schedule from the log.

However, do you see multiple 'Saved database' entries with timestamps 1 second apart?

Is your config.db writeable and OK? This is where the next rollover moment should be written to. You might try deleting that file so that Mnemosyne can recreate it.

**Comment by ADFairbanks** [comment]
I tried to reproduce it myself and it didn't happen. It probably only occurs during the extra gained hour of DST, which just so happens to be when I tried to use the program. I'm not sure if it's worth fixing for such a small time of use in that case. If you want to close it for being such a minor issue that's fine.
My data was still fine so that's not an issue, and there was indeed a backup database being created every second.
